Makefile

```
@@ -12,5 +12,17 @@ run:
 show:
 	open http://localhost:8080
 
+### DEV ###
+export CLEVER_DEVICES_KEY := DEV
+export CLEVER_DEVICES_IP := DEV
+
+mock:
+	cd mock_bustime_server && go build && ./server
+	
+dev:
+	npm install
+	go build
+	./nola-transit-map || echo "Couldn't run the binary."
```

```suggestion
	./nola-transit-map DEV=1 || echo "Couldn't run the binary."
```

Nit.  `make dev DEV=1` just feels smurfy to me.

---

Oddly, I could not get Make to set env vars inline with a shell command, so I used Make's syntax to export the mock API_KEY and IP env vars. I believe this sets those env vars for all commands in the Makefile, so users with API key might have to manually set env vars before running the server. I'm sure there's a better way that I'm unaware of, though.

---

I ended up removing the global env var exports from the Makefile and manually executed a shell command to set the env vars inline with executing the binary. I really expected to be able to simply declare those vars inline, but the binary would not see the env vars that way.